services/worker/src/worker/job_runners/config/split_names.py

```diff
@@ -87,6 +94,10 @@ def compute_split_names_from_streaming_response(
             f"Cannot get the split names for the config '{config}' of the dataset.",
             cause=err,
         ) from err
+
+    if not split_name_items:
+        raise SplitsEmptyError("The config has no splits.")
```

Comment:
What is the reason for a config to have an empty list of splits?

It would help enhance the error message to help the user fix the issue. Currently ("The config has no splits.") it's hard to take action based on it.

Reply:
> the README file does not include a split definition

but if no split is defined, we should have one split, named "train", by default, no?

Reply:
I am not a fan of assigning default values for user responses; I honestly like to fail on the first try (from-info) and leave the second try to get the correct split (I am unsure if it will always be 'train').

Reply:
sure, I meant: why can config-info have an empty list of splits?

Reply:
Thanks for helping me clarify; I think I found the real issue:
Some files have 0 rows, like https://huggingface.co/datasets/Vinay573/aarkoocustomdataset/tree/main/data?show_file_info=data%2Ftrain-00000-of-00001.parquet, and when trying to fill builder info https://github.com/huggingface/dataset-viewer/blob/main/services/worker/src/worker/job_runners/config/parquet_and_info.py#L692, it cleans all the previous content (even if it was possible to get the split before). Then, since it is 0 rows, no new split is added here: https://github.com/huggingface/dataset-viewer/blob/main/services/worker/src/worker/job_runners/config/parquet_and_info.py#L747.
I think the fix should be to add the split anyway but set the values to zero (as it is by default from the library):
![image](https://github.com/huggingface/dataset-viewer/assets/5564745/52b190c8-f8e1-48b3-b3d8-d9b71927261a)